INSTALL.md

```diff
@@ -378,7 +378,13 @@ echo "source ~/.bash_completion/alacritty" >> ~/.bashrc
 
 #### Fish
 
-To install the completions for fish, run
+Make sure you are in fish shell or run
+
+```
+fish
+```
+
+Once you are in fish shell, to install the completions for fish, run
```

Comment:
Maybe something like that? At least it should give you an idea how to adjust it, but the suggestion should be oneline imho.

```suggestion
To install the completions for fish, from inside the fish shell, run
```